File: core/src/test/java/ru/endlesscode/rpginventory/misc/TestConfiguration.java
package ru.endlesscode.rpginventory.misc;

import ru.endlesscode.rpginventory.configuration.Configurable;

public class TestConfiguration extends Configurable {

    private String aString = "Lorem ipsum dolor sit amet.";
    private int anInt = 5;

    @Override
    public String getHeader() {
        return null;
    }

    @Override
    public String getNodeName() {
        return "TestNode";
    }

    public String getaString() {
        return aString;
    }

    public void setaString(String aString) {
        this.aString = aString;
    }

    public int getAnInt() {
        return anInt;
    }

    public void setAnInt(int anInt) {
        this.anInt = anInt;
    }
}

File: core/api/src/test/java/ru/endlesscode/rpginventory/misc/TestConfiguration.kt
package ru.endlesscode.rpginventory.misc

import ru.endlesscode.rpginventory.configuration.Configurable

class TestConfiguration : Configurable {

    var aString = "Lorem ipsum dolor sit amet."
    var anInt = 5

    override fun getHeader(): String? = null

    override fun getNodeName(): String = "TestNode"
}
